src/context_store/storage/factory.py

```
@@ -298,6 +298,77 @@ async def create_storage(
+        import os
+
+        db_path = os.path.expanduser(settings.sqlite_db_path)
+        storage._outbox_writer = SqliteOutboxWriter()  # type: ignore[attr-defined]
+        reader = SqliteOutboxReader(db_path=db_path)
+    elif settings.storage_backend == "postgres":
+        storage._outbox_writer = PostgresOutboxWriter()  # type: ignore[attr-defined]
+        reader = PostgresOutboxReader(pool=storage._pool)  # type: ignore[attr-defined]
+    elif settings.storage_backend == "supabase":
+        storage._outbox_enabled = True  # type: ignore[attr-defined]
+        # Supabase は asyncpg を直接使えないため、Task 1.3 で定義した RPC を
+        # 呼び出す SupabaseOutboxReader を使用する（Python 実装は Task 4.1
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)